config_converter/config_mapper/config_mapper.py

```diff
@@ -171,17 +175,19 @@ def _convert_in_tail(d: config_pb2.Directive, stats: dict) -> dict:
             print(f'{p.name} is an unknown field.')
     for nd in d.directives:
         if nd.name == 'parse':
-            old_attrs = stats['attributes_recognized']
+            current_attribute_count = stats['attributes_recognized']
             for np in nd.params:
                 if np.name in special_fields:
                     _convert_parse_dir(fields, np)
                     stats['attributes_recognized'] += 1
                 else:
                     stats['attributes_unrecognized'] += 1
-            current_dir_attrs: int = get_num_attrs(nd)
-            if old_attrs + current_dir_attrs == stats['attributes_recognized']:
+            present_dir_attribute_count: int =\
+                    _get_aggregated_num_attributes(nd)
+            if current_attribute_count + present_dir_attribute_count ==\
```

Comment:
```suggestion
            if stats['attributes_recognized'] == current_attribute_count + present_dir_attribute_count
```

I'll put `stats['attributes_recognized']` in the beginning so it is consistent checking the same variable for 3 conditions.

Reply:
Yeah it does look clearer - will do!